app/models/katello/host/content_facet.rb

```diff
@@ -105,6 +105,14 @@ def image_mode_host?
         bootc_booted_image.present?
       end
 
+      def yum_or_yum_transient
+        if image_mode_host?
+          'yum --transient'
```

Comment:
Would it be more future proof to switch to `dnf`? I was under the impression that the yum-to-dnf symlink was supposed to get dropped eventually

Reply:
(the transient mode ones can be dnf at least)